boxoffice/templates/boxoffice.html

```diff
@@ -39,18 +39,23 @@ <h1 class="category-heading">{{title}}</h1>
                       </div>
                     </div>
                     <div class="ticket-price-qty">
-                      {{#if item.quantity_available > 0 && item.quantity_available < 5}}
+                      {{#if line_item.quantity_available > 0 && line_item.quantity_available < 5}}
                         <div class="ticket-description">
                           <p class="few-ticket">Only a few tickets left</p>
                         </div>
                       {{/if}}
-                      {{#if item.quantity_available > 0}}
-                        <p class="ticket-price"><i class="fa fa-inr rupee-sign"></i>{{ line_item.base_price }}</p>
+                      {{#if line_item.quantity_available > 0}}
+                        <div class="ticket-price-wrapper">
+                          {{#if line_item.unit_final_amount >= 0 && line_item.quantity === 0}}
+                            <p class="discount-price ticket-price"><i class="fa fa-inr rupee-sign"></i>{{ line_item.unit_final_amount }}</p>
+                          {{/if}}
+                          <p class="ticket-price {{#if line_item.unit_final_amount >= 0 && line_item.quantity === 0}}strike{{/if}}"><i class="fa fa-inr rupee-sign"></i>{{ line_item.base_price }}</p>
```

Comment:
If check is done this way, when the discounted item is removed from the cart, there is latency and transition isn't smooth.
<img width="1434" alt="screen shot 2016-06-08 at 2 27 02 pm" src="https://cloud.githubusercontent.com/assets/11267213/15888661/605cf364-2d85-11e6-877c-e647e11fc2a5.png">